test/integration/cdn/test_application.rb

```diff
@@ -10,7 +10,7 @@
     @hanami_env       = ENV['HANAMI_ENV']
     ENV['HANAMI_ENV'] = 'production'
 
-    assert system("HANAMI_ENV=production bundle exec hanami assets precompile")
+    system("HANAMI_ENV=production bundle exec hanami assets precompile")
```

Comment:
Here we're asserting that the returning value of `Kernel.system` is `true`. This specific returning value means that the system call is successful and so we can tell that the assets precompilation went good.

Please restore this assertion. 😄